tests/test_naver_map_crawler.py

```
@@ -0,0 +1,107 @@
+def test_get_random_ua_installed(mock_installed_packages):
+    """fake-useragent 설치 시 랜덤 UA 반환 검증"""
+    _ = mock_installed_packages
+    crawler = NaverMapCrawler(headless=True)
+    ua = crawler._get_random_ua()
+    assert ua == "Mocked/UserAgent 1.0"
+
+def test_get_random_ua_generic_exception(mock_installed_packages):
+    """fake-useragent 사용 중 일반 예외(Exception) 발생 시 Fallback UA 반환 검증"""
+    mock_ua_instance, _ = mock_installed_packages
+    # random 접근 시 예외 발생 시뮬레이션
+    type(mock_ua_instance).random = property(lambda self: (_ for _ in ()).throw(Exception("Random Error")))
```

Ruff (0.15.0)

[warning] Unused lambda argument: `self`

(ARG005)